Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,43 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import NumberButton from "./components/ButtonComponents/NumberButton";
+import ActionButton from "./components/ButtonComponents/ActionButton";
+import btnContainer from "./components/ButtonComponents/btn-container";
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="app">
+      {/* <DisplayComponent />
+          <ButtonComponent /> */}
+      <CalculatorDisplay />
+      <btnContainer className="button-container">
+        <ActionButton className="text" text="clear" />
```

Comment:
this isn't very DRY. Think about how you could implement the map feature to render your buttons. What styles would you need to apply to separate the Number buttons and operator buttons into different components?